core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -48,10 +48,18 @@
       "Fully qualified class name of the class that implements the CryptoModule interface, to be used in setting up encryption at rest for the WAL and "
           + "(future) other parts of the code."),
   @Experimental
-  CRYPTO_CIPHER_SUITE("crypto.cipher.suite", "NullCipher", PropertyType.STRING, "Describes the cipher suite to use for the write-ahead log"),
+  CRYPTO_CIPHER_SUITE("crypto.cipher.suite", "NullCipher", PropertyType.STRING,
```

Comment:
Could possibly changes this to `crypto.file.cipher.suite`.  But maybe a lot of other props would need to change.  I thought of this when looking at the `crypto.wal.cipher.suite` property.